public_html/layouts/basic/modules/Vtiger/resources/Inventory.js

```diff
@@ -1475,23 +1475,22 @@ $.Class(
 		 */
 		registerClearReferenceSelection() {
 			this.form.on('click', '.clearReferenceSelection', e => {
-				const row = this.getClosestRow($(e.currentTarget));
-				this.removeSubProducts(row);
-				row
-					.find('.unitPrice,.tax,.discount,.margin,.purchase,.js-tax-percent')
-					.val(App.Fields.Double.formatToDisplay(0));
-				row.find('.qty').val(1);
-				row.find('textarea,.valueVal').val('');
-				row.find('.valueText').text('');
-				row.find('.qtyParamInfo').addClass('d-none');
-				row
-					.find('.recordLabel')
-					.val('')
-					.removeAttr('readonly');
-				if (!this.isGroupTaxMode()) {
-					this.setTaxParam(row, []);
+				if(referenceGroup.length) {
```

Comment:
Undefined variable 'referenceGroup'

Reply:
lost in copying ... added referenceGroup var